### ISSUE TITLE
Investigate why the ejb32 test count is so high

### ISSUE THREAD

**Issue by starksm64** [jakartaee/platform-tck#1475]
A run of the current ejb32 module with no keyword tags shows 1322 tests. The latest run of glassfish on EE10 shows 825 tests:
https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/10.0.x/141/artifact/ejb32-results.tar.gz

We need to figure out how there are 50% more tests availble.

**Comment by starksm64** [comment]
The EE10 ejb32 com.sun.ts.tests.ejb32.lite.timer.basic.concurrency test package has the following tests:

```
 name="lookupTimerService_from_ejbembed"
 name="lookupTimerService_from_ejblitejsp"
 name="lookupTimerService_from_ejbliteservlet"
 name="lookupTimerService_from_ejbliteservlet2"
 name="writeLockTimeout_from_ejbembed"
 name="writeLockTimeout_from_ejblitejsp"
 name="writeLockTimeout_from_ejbliteservlet"
 name="writeLockTimeout_from_ejbliteservlet2"
 name="lookupTimerService_from_ejblitejsf"
 name="writeLockTimeout_from_ejblitejsf"
```

We are generating:
```
ClientEjblitejsfTest
ClientEjblitejspTest
ClientEjbliteservlet2Test
ClientEjbliteservletTest
JsfClientEjblitejsfTest
JsfClientEjblitejspTest
JsfClientEjbliteservlet2Test
JsfClientEjbliteservletTest
```

So the issue is the JsfClient EE10 class is that base class for the ejblitejsf vehicle, and there should only be a JsfClientEjblitejsfTest generated. The JsfClientEjblitejspTest, JsfClientEjbliteservlet2Test and JsfClientEjbliteservletTest should not be generated. The ClientEjblitejsfTest should also not be generated.

**Comment by starksm64** [comment]
Fixed by the 1.0.0-M9 release of jarkataee-tck-tools
